### DISCUSSION TITLE
主题中图片查看器是不是少了一些功能

### DISCUSSION THREAD

**Discussion by wungjyan** [ppoffice/hexo-theme-icarus#975] [Q&A]
看文档是用了lightGallery， 应该是支持以下图中的功能的：
![image](https://user-images.githubusercontent.com/29590076/144015731-4e5a9563-d8ce-4ab2-b01d-82a57a07f309.png)

但是实际上主题图片查看器，只有下载功能了：
![image](https://user-images.githubusercontent.com/29590076/144015834-c85f109a-fb95-4163-b186-83090275a19b.png)

请问怎么可以支持完整的功能

**Comment by ppoffice** [comment]
@wungjyan 需要自行在layout中添加lightgallery的相关插件：
https://www.lightgalleryjs.com/docs/getting-started/#include-css-and-javascript-files
